test/export/ExtensionExporter.test.ts

```diff
@@ -3,17 +3,17 @@ import { FSHTank, FSHDocument } from '../../src/import';
 import { FHIRDefinitions, load } from '../../src/fhirdefs';
 import { Extension } from '../../src/fshtypes';
 import { logger } from '../../src/utils/FSHLogger';
+import { LoggerSpy } from '../testhelpers/loggerSpy';
 
 describe('ExtensionExporter', () => {
   let defs: FHIRDefinitions;
   let doc: FSHDocument;
   let input: FSHTank;
   let exporter: ExtensionExporter;
-  let mockWriter: jest.SpyInstance<boolean, [any, string, ((error: Error) => void)?]>;
+  const loggerSpy = new LoggerSpy(logger);
```

Comment:
Since `logger` itself is a global, is it necessary to pass it in to a constructor for `LoggerSpy`?  If not, then you could have `testhelpers/loggerSpy` just export an _instance_ of `loggerSpy` that is already initialized with the global logger.  Then you wouldn't need this extra line of code here.

Reply:
Sure, I suppose I could do that instead!

Reply:
This should be done and ready to go.